<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtobusBilgiSistemi</class>
 <widget class="QMainWindow" name="OtobusBilgiSistemi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OtobusBilgiSistemi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="tbIsletme">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
icon-align:center </string>
    </property>
    <property name="text">
     <string>isletme</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSefer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sefer</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbOtobus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>otobus</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSeyehatKurallari">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>kural</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSeyehat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>seyehatler</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Microsoft Sans Serif&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>KUTULARIN İÇİ KAFASINA GÖRE 0 VEYA 99 OLUYOR,
 AÇIP KAPATINCA DEĞİŞİYOR HEP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_letme">
    <property name="title">
     <string>İşletme</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOtobusler">
    <property name="title">
     <string>Otobusler</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSeferler">
    <property name="title">
     <string>Seferler</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSeyehat_Kurallar">
    <property name="title">
     <string>Seyehat Kuralları</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSeyehatler">
    <property name="title">
     <string>Seyehatler</string>
    </property>
   </widget>
   <addaction name="menu_letme"/>
   <addaction name="menuOtobusler"/>
   <addaction name="menuSeferler"/>
   <addaction name="menuSeyehat_Kurallar"/>
   <addaction name="menuSeyehatler"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarVeriGiris">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
